src/cosmic_ray/commands/init.py

```diff
@@ -56,7 +57,27 @@ def init(module_paths, work_db, config):
       work_db: A `WorkDB` instance into which the work orders will be saved.
       config: The configuration for the new session.
     """
-    operator_names = cosmic_ray.plugins.operator_names()
+
+    operator_names = list(cosmic_ray.plugins.operator_names())
+
+    if log.isEnabledFor(logging.INFO):
```

Comment:
Why do we need this here? Isn't this what the "operators" command is for?

Reply:
With the exception of this logging, I think this PR looks good.

Reply:
I don't understand the problem:
- Just this test? It is useless to build complex log trace is there is no log watchers.
- The entire log block: It's just cosmetic, but to see and understand all available operators, some classification seems useful.

Reply:
My point is that we've already got the `cosmic-ray operators` command which does this. If you want to update the output format of `operators`, I'm open to that discussion. But this logging just seems to duplicate an existing feature unnecessarily.

Reply:
Ok , I have forgotten `cosmic-ray operators`. I will rollback this log